src/server.c

```diff
@@ -267,7 +267,7 @@ void exitFromChild(int retcode) {
 /*====================== Hash table type implementation  ==================== */
 
 /* This is a hash table type that uses the SDS dynamic strings library as
- * keys and redis objects as values (objects can hold SDS strings,
+ * keys and valkey objects as values (objects can hold SDS strings,
```

Comment:
Since we already renamed "redisObject" to "serverObject", perhaps "server objects" here or just "objects"?

Reply:
Looks unstable is already updated as you suggested.